CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Device control library change log
 =================================
 
+4.2.1
+-----
+
+  * FIXED: Some errors were being printed to stdout, now all use 
+    stderr. Common format also adopted
```

Comment:
The change log checker is complaining about this line.

Reply:
Fixed by #104